src/stories/2-frontend-engine/validation/validation-story-component.tsx

```diff
@@ -132,8 +132,8 @@ export const ValidationStoryComponent = ({
 	// =============================================================================
 	return (
 		<Wrapper>
-			<V2_Layout.Content type="grid">
-				<V2_Layout.ColDiv desktopCols={12} tabletCols={8} mobileCols={4}>
+			<Layout.Content type="grid">
+				<Layout.ColDiv xlCols={12} mdCols={8} xxsCols={4}>
```

Comment:
as xxs-md are now 8 columns, the proportion needs to be adjusted

the intention of the columns here were to take up the full grid, so it should be 12 columns for lg-xxl and 8 columns for xxs-md

this can be expressed with `xxsCols={9}` and `lgCols={12}`